CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Intermediate bytes of CSI sequences not checked
 - Wayland clipboard integration
 - Use text mouse cursor when mouse mode is temporarily disabled with shift
+- Wayland primary clipboard storing when stopping selection outside of the window
```

Comment:
I think instead of saying we fixed that the clipboard stores the selection when stopping outside the window, I think it makes more sense to phrase this a bit differently:

```
- Wayland primary selection clipboard not storing text when selection is stopped outside of the window
```